flake.nix

```diff
@@ -3,33 +3,36 @@
   
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
+    circom-compat.url = "github:codex-storage/circom-compat-ffi";
```

Comment:
I did try it with relative path, but the problem is the `.?submodules=1` part when building the derivation. Since we declare the `circom-compat` flake as input when the submodules are not cloned yet, the build fails because the flake file is not present yet on the filesystem. Because of that I am using the GitHub URL here.